src/gennylib/include/gennylib/v1/PoolManager.hpp

```diff
@@ -129,7 +129,16 @@ class PoolManager {
     /**
      * Obtain a connection or throw if none available.
      *
-     * This is allowed to be called from multiple threads simultaneously.
+     * This connection will be pre-warmed by default, unless "NoPreWarm" is set to true, e.g.:
+     * 
+     * ```yaml
+     * Clients:
+     *   Default:
+     *     NoPreWarm: true
```

Comment:
nit: IMO, "PreWarm: true/false" is more straightforward than "NoPreWarm: false/true"